lib/dialects/mssql/query-generator.js

```diff
@@ -685,6 +685,11 @@ var QueryGenerator = {
     if (options.limit || options.offset) {
       if (!options.order || (options.include && !orders.subQueryOrder.length)) {
         fragment += (options.order && !isSubQuery) ? ', ' : ' ORDER BY ';
+
+        if(model.name){
+          fragment += this.quoteIdentifier(model.name) + '.';
+        }
```

Comment:
Understood well. I will update my code in about 6 days


> On Jul 3, 2016, at 7:41 PM, Ruben Bridgewater <notifications@github.com> wrote:
> 
> In lib/dialects/mssql/query-generator.js:
> 
> > @@ -685,6 +685,11 @@ var QueryGenerator = {
> >      if (options.limit || options.offset) {
> >        if (!options.order || (options.include && !orders.subQueryOrder.length)) {
> >          fragment += (options.order && !isSubQuery) ? ', ' : ' ORDER BY ';
> > +
> > +        if(model.name){
> > +          fragment += this.quoteIdentifier(model.name) + '.';
> > +        }
> There is always a model.name. So you do not have to test for this. And as you want to add the table name you should use .quoteTable(model). That way also schemas are supported.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>